src/System.Private.CoreLib/src/System.Private.CoreLib.csproj

```diff
@@ -797,7 +786,7 @@
     <Compile Include="..\..\Common\src\Interop\Windows\mincore\Interop.SetLastError.cs">
       <Link>Interop\Windows\mincore\Interop.SetLastError.cs</Link>
     </Compile>
-    <Compile Include="..\..\Common\src\Interop\Windows\mincore\Interop.Globalization.cs">
+    <Compile Condition="'$(EnableDummyGlobalizationImplementation)' != 'true'" Include="..\..\Common\src\Interop\Windows\mincore\Interop.Globalization.cs">
```

Comment:
Done, think I got them all. (The Unix ones in a separate commit.)

I am wondering what testing is appropriate here - I can verify that the CoreCLR top200 all pass on Win32 with EnableDummyGlobalizationImplementation on, I'm still getting working a Unix test setup.